### PR TITLE
Use anonymous GCS client only for custom dc and local user data paths.

### PR DESCRIPTION
**Pull request by keyurva**

### CODE REVIEW

File: nl_server/util.py [hunk 1]
```diff
@@ -14,6 +14,29 @@
 
 import os
 
+from shared.lib.gcs import is_gcs_path
+
+_IS_CUSTOM_DC_ENV_VAR = 'IS_CUSTOM_DC'
```
Comment (pradh) [inline nl_server/util.py:19]: Why use constant if not referenced >1 time?  Adds more hops for the reader of code :)

**Review by pradh — APPROVED** [review]
Thanks

File: nl_server/util.py (hunk 1 above)
Comment (keyurva) [reply nl_server/util.py]: I prefer to declare instead of inlining constants but have removed it here.

**Comment by keyurva** [comment]
Thanks for the review.

**Pull request merged**